python/ray/includes/unique_ids.pxi

```diff
@@ -135,6 +136,17 @@ cdef class ObjectID(BaseID):
     def __init__(self, id):
         check_id(id)
         self.data = CObjectID.FromBinary(<c_string>id)
+        worker = ray.worker.global_worker
+        # TODO(edoakes): a few ObjectIDs are created before the core worker is
```

Comment:
Which ones are these? Theoretically it shouldn't be possible to submit tasks or put objects before ray is initialized.

Reply:
Ah it might be the object id that warms up the object store. That one should be ok and can get lost (maybe make sure that's indeed the only one and edit the comment to include this insight).

Reply:
Fixed this, it was errantly used in the monitor.